tableau_utilities/tableau_file/tableau_file_objects.py

```diff
@@ -982,6 +988,7 @@ def dict(self):
 @dataclass
 class Extract(TableauFileObject):
     object_id: str = None
+    user_specific: bool = None
```

Comment:
@jaybythebay Since this is a bool, can you add a `self.__to_bool('user_specific')` line to the `__post_init__` of `TableauFileObject`?
This is just used to enforce that the attribute is set as a `bool`, if it is read as a `str`.

Reply:
@JustinGrilli  added